### PR TITLE
feat(package): add update notifier

### PR DESCRIPTION
**Pull request by levithomason**
Once a day, DevLab will check for updates and notify the user on next run:

https://www.npmjs.com/package/update-notifier

***

image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -16,6 +19,9 @@ const tmpdir = require('./tempdir')()
 global.instanceId = require('shortid').generate()
 
 const instance = {
+  checkForUpdates: () => {
+    updateNotifier({ pkg }).notify()
```
Comment (Fluidbyte) [inline src/index.js]: This can probably just be ignored unless you really want to stub it...

Comment (levithomason) [inline src/index.js]: Yep, I think coverage here is mandatory.

Comment (Fluidbyte) [inline src/index.js]: Sorry, meant `istanbul ignore`'d...

**Comment by levithomason** [comment]
Covered it.  This was the 3rd sandbox I was about to instantiate, so I pulled it into `test/sandbox.js` instead.  It isn't a magic global, but we can explicitly import it when needed.  Small step in the right direction.

Because I was updating `index.spec.js` with a sandbox test, I converted that file to use the sandbox entirely.

**Comment by Fluidbyte** [comment]
Clinton Approves

**Comment by levithomason** [comment]
Switched to proxyquire for `checkForUpdates` assertions.  Thanks for the tip @Fluidbyte.

File: test/src/index.spec.js [hunk 2]
```diff
@@ -1,58 +1,61 @@
 const path = require('path')
 const fs = require('fs')
 const Promise = require('bluebird')
+
+const sandbox = require('test/sandbox')
+const mockUpdateNotifierInstance = { notify: sandbox.spy() }
+const mockUpdateNotifier = sandbox.spy(() => mockUpdateNotifierInstance)
+proxyquire('src/index', {
```
Comment (Fluidbyte) [inline test/src/index.spec.js]: 👍

**Pull request merged**